apps/web/src/app/[locale]/(logged-in)/insights/query-string-util.ts

```diff
@@ -9,6 +9,11 @@ import {
 
 export type OrderType = 'asc' | 'desc';
 
+export type QueryParamsType = {
+  key: string;
```

Comment:
```suggestion
    key: SearchParamsKeys;
```
I like narrow typing, but maybe I this could create issues down the line?

Reply:
I think the URL will be used for more than searching, so I think we should keep it as a string for now.